app/javascript/packs/application.js.erb

```diff
@@ -0,0 +1,31 @@
+import "core-js/stable"
+import "regenerator-runtime/runtime"
```

Comment:
What causes us to need regenerator? Is it due to supporting pre-Promise browsers, or something like that? If so, I think I'd like to see it loaded as a Babel transform e.g. https://babeljs.io/docs/en/babel-plugin-transform-regenerator instead. I'd love to know more. 

I'd also love to commit firmly to which browser versions we support, and then perhaps test the app with them, and certainly if we're doing something for IE11, I'd like clarity on that so I don't accidentally remove that down the road.

Reply:
honestly, it was in a tutorial of how to move from using sprockets to webpacker and I didn't think too much about it